README.md

```diff
@@ -1,37 +1,43 @@
 
- The​ ​app​ ​has​ ​to​ ​allow​ ​us​ ​to​ ​access​ ​a​ ​lobby,​ ​view​ ​players’​ ​details,​ ​create​ ​a​ ​game​ ​where​ ​you​ ​invite others​ ​as​ ​well​ ​as​ ​rearrange​ ​teams​ ​and​ ​then,​ ​after​ ​a​ ​match,​ ​one​ ​can​ ​add​ ​some​ ​stats,​ ​which​ ​will be​ ​stored​ ​for​ ​later.
+ At a minimum, the​ ​app​ should ​allow​ ​us​ ​to​ ​access​ ​a​ ​lobby and​ ​create​ ​a​ ​game​ ​where​ ​you​ ​invite others​. If you want to and have enough time you can add bonus features: ​players’​ ​details view, ​rearranging ​teams​ or,​ ​after​ ​a​ ​match,​ ​add​ing and storing ​some​ ​stats.
 
- We​ ​ask​ ​you​ ​not​ ​to​ ​spend​ ​​more​ ​than​ ​4​ ​hours​​ ​for​ ​this​ ​task.​ ​If​ ​you​ ​don’t​ ​manage​ ​to​ ​implement everything,​ ​no​ ​worries​ ​-​ ​we’d​ ​be​ ​very​ ​pleasantly​ ​surprised​ ​if​ ​you​ ​did.
+ We​ ​ask​ ​you​ ​not​ ​to​ ​spend​ ​​more​ ​than​ ​4​ ​hours​​ ​for​ ​this​ ​task.​ Don't worry if ​you​ ​don’t​ ​manage​ ​to​ ​implement a gazillion of features or breathtaking design -​ focus on the create game and lobby functionalities. We'll appreciate clean elegant code, a thoughtful solution and a nice project structure!
  
  # The​ ​functionality
  - Lobby​ ​(with​ ​a​ ​list​ ​of​ ​players​ ​and​ ​some​ ​information) 
- - Detailed​ ​player​ ​view
  - Create​ ​a​ ​game 
     - Invite​ ​players
-    - Rearrange​ ​teams​ ​by​ ​reassigning​ ​players 
-    - Validation​ ​(teams​ ​can​ ​only​ ​consist​ ​of​ ​4​ ​or​ ​2​ ​players)
+
+
+## Nice to haves
 - Mark​ ​which​ ​team​ ​won 
-## Extra​ ​brownie​ ​points​ ​ideas 
+- Rearrange​ ​teams​ ​by​ ​reassigning​ ​players 
+- Detailed​ ​player​ ​view
 - Add​ ​some​ ​stats​ ​to​ ​each​ ​player
 - Save​ ​stats
- - Login​ ​(can​ ​be​ ​mocked)
- - Visually​ ​pleasing​ ​design​ ​and​ ​interactions
+- Login​ ​(can​ ​be​ ​mocked)
+- Visually​ ​pleasing​ ​design​ ​and​ ​interactions
  
- # Look​ ​and​ ​feel 
- We​ ​provide​ ​no​ ​design,​ ​bar​ ​a​ ​few​ ​(deliberately)​ ​ugly​ ​wireframes.​ ​Any​ ​aesthetic​ ​decisions​ ​are​ ​up to​ ​you;​ ​obviously,​ ​any​ ​effort​ ​put​ ​in​ ​this​ ​area​ ​will​ ​be​ ​appreciated​ ​and​ ​taken​ ​into​ ​account. The​ ​only​ ​requirement​ ​is:​ ​​it​ ​has​ ​to​ ​be​ ​responsive​ ​​and​ ​work​ ​well​ ​on​ ​a​ ​variety​ ​of​ ​platforms, browsers​ ​(the​ ​lowest​ ​we​ ​support​ ​is​ ​IE11),​ ​devices. 
+ ## Look​ ​and​ ​feel 
+ We​ ​provide​ ​no​ ​design so any​ ​aesthetic​ ​decisions​ ​are​ ​up to​ ​you. The​ ​only​ ​requirement​ ​is:​ ​​it​ ​has​ ​to​ ​be​ ​responsive​ ​​and​ ​work​ ​well​ ​on​ ​a​ ​variety​ ​of​ ​platforms, browsers​ ​(the​ ​lowest​ ​we​ ​support​ ​is​ ​IE11),​ ​devices. 
```

Comment:
does it have to be responsive? 
it takes time to write responsive css,
also chcecking if it works on all browsers is time consuming. 
last but not least I don't have IE11 on my machine. it would be a problem for me to check it as I don't own a device with IDE setup and Windows

Reply:
Sure. Moved responsiveness to the `optional` list.